src/pat/auto-suggest/auto-suggest.js

```diff
@@ -50,6 +51,7 @@ export default Base.extend({
         this.options = parser.parse(this.el, this.options);
 
         let config = {
+            separator: this.options.valueSeparator,
```

Comment:
I do not get why we have a `separator` and a `tokenSeparator`. I hope it's fine like that :)

Reply:
initially I thought we need to modify the `tokenSeparator`. But not, it's for the tagging feature - when you add the tokenSeparator a new tag/token will be created. See:
https://select2.github.io/select2/#doc-tokenSeparators